src/app/extensions/quoting/models/quoting/quoting.interface.ts

```diff
@@ -55,3 +55,7 @@ export interface QuoteRequestResponse extends QuoteBaseData<Link | QuoteRequestL
 }
 
 export type QuoteData = Link | QuoteResponse | QuoteRequestResponse;
+
+export interface QuoteEntitiesOptions {
```

Comment:
Interfaces should only be used to accept data of REST calls. Anyway, I 'd prefer to remove it at all.
If use use an interface only at one place you can define it at this place, otherwise in the model.